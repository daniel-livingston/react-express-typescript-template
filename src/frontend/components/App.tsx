import * as React from "react";

export class App extends React.Component<{}, {}> {
	render() {
		return (
			<div>
				<h1>This is a React-Express-Typescript template.</h1>
				<h2>It also uses Sass</h2>
			</div>
		);
	}
}

export default App;
