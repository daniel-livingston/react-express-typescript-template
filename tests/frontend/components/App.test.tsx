import * as React from "react";
import TestRenderer from "react-test-renderer";
import App from "../../../src/frontend/components/App";

test("renders snapshot correctly", () => {
	const wrapper = TestRenderer.create(<App />);

	expect(wrapper.toJSON()).toMatchSnapshot();
});
