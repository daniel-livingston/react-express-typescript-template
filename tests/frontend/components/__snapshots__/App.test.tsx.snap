// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div>
  <h1>
    This is a React-Express-Typescript template.
  </h1>
  <h2>
    It also uses Sass
  </h2>
</div>
`;
